### DISCUSSION TITLE
Clean Install Windows or Linux ?

### DISCUSSION THREAD

**Discussion by AlexUrrutia** [lllyasviel/Fooocus#2589] [General]
I'm setting up a dedicated PC to run **Fooocus**, I am wondering what could be the best **OS** choice to run it, **Windows** or **Ubuntu**?
**AMD Ryzen 5 3400G, 16GB RAM, RTX3060**, what do you guys think?

**Comment by Rogala** [comment]
Install the system you know best. So that you don't have to deal with minor tasks in the fooocus installation environment.
Better 32gb of RAM, I have at 16 sometimes shortage when creating with 3 Loras or upscale in on 2x.

**Comment by mashb1t** [comment]
@AlexUrrutia both are equally performing considering generation speed, so feel free to choose which ever you prefer more.

**Comment by GuilhermeNobrega** [comment]
Hi @AlexUrrutia! You can use both at the same time. In my case, I like to use WSL (Windows Subsystem for Linux), I have Ubuntu, Kali... it's simpler than a virtual box. try it, is very easy to install:

https://ubuntu.com/desktop/wsl
